### ISSUE TITLE
consider archiving

### ISSUE THREAD

**Issue by tannerblair** [NIVeriStandAdd-Ons/Easy-Custom-Device-Tool#2]

**Comment by AndrewHeim** [comment]
We can archive this, but should note that it is still a good tool for a learner to quickly put together a simple custom device.

It would be nice to clarify guidelines on what kinds of custom devices are easily accomplished and what kind are not.  There are some assumptions made in its code that makes some things simple (while precluding others).

**Comment by tannerblair** [comment]
Agreed Andrew. I think there is room for it or something like it for sure, and we should keep it in mind for a potential future investment
